README.md

```diff
@@ -46,21 +53,21 @@ Usage: dandi [OPTIONS] COMMAND [ARGS]...
 Options:
   --version
   -l, --log-level [DEBUG|INFO|WARNING|ERROR|CRITICAL]
-                                  Log level name  [default: INFO]
-  --pdb                           Fall into pdb if errors out
+                                  Log level name  [default: INFO].
+  --pdb                           Fall into pdb if errors out.
   --help                          Show this message and exit.
 
 Commands:
-  download  Download a file or entire folder from DANDI
+  download  Download a file or entire folder from DANDI.
   ls        List .nwb files and dandisets metadata.
   organize  (Re)organize files according to the metadata.
-  register  Register a new dandiset in the DANDI archive
+  register  Register a new dandiset in the DANDI archive.
```

Comment:
The `register` command is no more, and some additional commands have been added.  You may want to update the `dandi` and `dandi ls --help` output.

Reply:
Thanks @jwodder, I installed DANDI, generated the output and updated accordingly.